player/public/coreplugins/org.ekstep.overlay-1.0/renderer/templates/menu.html

```diff
@@ -6,7 +6,7 @@
     </a>
     <a ng-if="showUser" class="gc-menu-btn" href="javascript:void(0)" ng-click="openMenu()">
         <img class="menu-icon icon" ng-src="{{imageBasePath}}menu_icon.png" ng-hide="menuOpened"/>
-        <img class="menu-icon user" ng-src="{{currentUser.profileImage}}" ng-hide="menuOpened"/>
+        <img class="menu-icon user" ng-src="{{imageBasePath}}avatar_anonymous.png" ng-hide="menuOpened"/>
```

Comment:
This is the callback for the mobile team when the player changes the user So that when user will close the player mobile will reload the app for that new user.